scripts/get_community_assigned_issues.py

```diff
@@ -0,0 +1,73 @@
+# Prints a list of links to all issues that are assigned
+# to external contributors.
+# This is not used by any workflows, and meant to be used locally
+# whenever need arises. The script requires `GITHUB_TOKEN`
+# environment variable with 'repo' and 'read:org' permissions. 
+
+import os
+import requests
+
+ORG = "learningequality"
+REPOS = ["kolibri", "studio", "kolibri-design-system", "le-utils", ".github","ricecooker"]
+
+TOKEN = os.getenv("GITHUB_TOKEN")
+if not TOKEN:
+    raise EnvironmentError("Please set the GITHUB_TOKEN environment variable with 'repo' and 'read:org' permissions.")
+
+BASE_URL = "https://api.github.com"
+HEADERS = {
+    "Authorization": f"token {TOKEN}",
+    "Accept": "application/vnd.github+json"
+}
+
+def get_team_members(org):
+    """Fetch all team members for the organization."""
+    team_members = set()
+    url = f"{BASE_URL}/orgs/{org}/members"
+    while url:
```

Comment:
I see that we are doing manual pagination here. I was curious to know if it would be helpful to use external libraries to handle this kind of stuff? Searching a bit I see quite a lot of them https://github.com/PyGithub/PyGithub and some third party suggested [once](https://docs.github.com/en/rest/using-the-rest-api/libraries-for-the-rest-api?apiVersion=2022-11-28#python) by Github. I know this might be an overkill for this but, just wanted to think about this if there's any further scope of development of the script.